zos_concepts/gdg_datasets/create_copy_submit/README.md

```diff
@@ -0,0 +1,87 @@
+# Create GDG, copy delete and submit of Generation Data Group
+This playbook demonstrates how to create a GDG create copy delete and submit
+creations of GDS.
+
+This playbook uses:
+  - collection:
+    - ibm.ibm_zos_core
+  - modules:
+    - zos_copy
+    - zos_dataset
+    - zos_job_submit
+
+It is a good practice to review the playbook contents before executing
+them. It will help you understand the requirements in terms of space, location,
+names, authority, and the artifacts that will be created and cleaned up.
+
+> [!IMPORTANT]
```

Comment:
See main comment, best to remove this section.

Reply:
I think we are past 1.10.0 long enough to remove this section, in fact not for you but we should go back and edit all playbooks eventually.

<img width="971" alt="image" src="https://github.com/user-attachments/assets/ea964e74-6c2f-4e56-9f37-2b8d2931cc63">